src/iomusxml.cpp

```diff
@@ -3695,8 +3695,11 @@ void MusicXmlInput::ReadMusicXmlPrint(pugi::xml_node node, Section *section)
     assert(section);
 
     if (node.attribute("new-page").as_bool()) {
-        Pb *pb = new Pb();
-        section->AddChild(pb);
+        int pageBreaks = node.attribute("blank-page").as_int() + 1;
```

Comment:
When a variable is used as a loop boundary, it is good to have is as `const` in order to prevent side effects
```suggestion
        const int pageBreaks = node.attribute("blank-page").as_int() + 1;
```

Reply:
That makes a lot of sense to me even though I'm only dipping a toe into C++.